docs/source/setup-dev.md

```diff
@@ -58,13 +58,13 @@ create a virtualenv to work in
         For each of them:
             - Select Py.test from the ‘Default test runner’
     - Press `Apply`
- 
+
 
 ## Detailed Setup
 
 ### Setup Python
 
-One needs Python 3.5 to work with the code. You can use `dev-setup/ubuntu/setup_dev_python.sh` script for quick installation of Python 3.5, pip 
+One needs Python 3.5 to work with the code. You can use `dev-setup/ubuntu/setup_dev_python.sh` script for quick installation of Python 3.5, pip
```

Comment:
I thought for Ubuntu 20.04, we had to bump to a version beyond Python 3.5.

Reply:
We did, it's 3.8.  Docs need to be updated.  cc @pSchlarb

Reply:
Part of this PR or will it follow.  I can approve if it is OK to leave this.

Reply:
@pSchlarb, To follow?

Reply:
will incorporate it to the documentation changes.